src/pages/welcome.js

```diff
@@ -20,18 +20,25 @@ export const welcomeScreen = async (auth, route) => {
     welcomeScreenTemplate(name || response.data.email, response.data, auth, route);
 }
 
-/*
-IMPORTANT - DO NOT PUSH THIS DEV CHANGE TO STAGE/PROD, HFHS SHOULD ONLY HAVE RESEARCH DASHBOARD ACCESS IN STAGE/PROD
-Removed 'HFHS' from researchSiteArray to give HFHS access to both research/clinical in dev only
-*/ 
 const clinicalSiteArray = ['KPNW', 'KPCO', 'KPHI', 'KPGA'];
-const researchSiteArray = ['MFC', 'UCM', 'HP', 'SFH'];
+const researchSiteArray = ['MFC', 'UCM', 'HP', 'HFHS','SFH'];
 
 const welcomeScreenTemplate = (name, data, auth, route) => {
     let template = '';
     let dashboardSelectionStr = '';
+    
+    if(location.host !== urls.stage || location.host !== urls.prod) {
+        clinicalSiteArray.push('HFHS');
+    }
 
-    if (clinicalSiteArray.includes(data.siteAcronym)) {
+    if (clinicalSiteArray.includes(data.siteAcronym) && researchSiteArray.includes(data.siteAcronym)) {
+        dashboardSelectionStr = `
+            <select required class="col form-control" id="dashboardSelection">
+                <option value="">-- Select Dashboard --</option>
+                <option value="clinical">Clinical Dashboard</option>
+                <option value="research">Research Dashboard</option>
+            </select>`;
```

Comment:
This condition check produces same result as the `else` condition, and these 2 parts have same code. Thus they can be combined.